chimedb/dataset/utils.py

```diff
@@ -0,0 +1,75 @@
+"""Dataset utils and click scripts."""
+
+import click
+
+from chimedb.core import connect as connect_db, close as close_db
+from chimedb.dataset.get import Dataset, DatasetCache, index
+
+
+@click.group()
+def cli():
+    """Click entry point."""
+    pass
+
+
+@cli.command()
+@click.argument("dataset_id")
+def treesize(dataset_id):
+    """Print number of nodes in a dataset tree."""
```

Comment:
```suggestion
    """Print number of nodes in the tree containing DATASET_ID"""
```

Reply:
I think you're typically supposed to mention the arguments when writing docstrings that end up as click help messages.